deployment/ccip/changeset/cs_update_rmn_config.go

```diff
@@ -70,25 +73,38 @@ func SetRMNRemoteOnRMNProxyChangeset(e deployment.Environment, cfg SetRMNRemoteO
 		return deployment.ChangesetOutput{}, fmt.Errorf("failed to get proposer MCMS for chains %v: %w", cfg.ChainSelectors, err)
 	}
 	var timelockBatch []timelock.BatchChainOperation
+	g := new(errgroup.Group)
```

Comment:
It will only create confusion if we handle parallelization in a singular CS like this. We are going to handle CS execution with deployer group which abstracts away MCMS and deployer key driven operation in CS - https://github.com/smartcontractkit/chainlink/blob/develop/deployment/ccip/changeset/deployer_group.go
The plan is to move to deployer group after this is tested throughly. You can either make this change in deployer group and add related test to it or if it's time-sensitive you can make parallel calls from the CS caller for diff chains.